english/invite-friends/camp_fire/ghost-stories/check_source_team/check_source_team.md

```diff
@@ -2,5 +2,7 @@ You and your friends go to the next room. Because you were the one to suggest th
 you see that a huge part of the marshmallow wall has been torn down. A trail of blood leads through the hole and your
 grandmother who had been watching tv is missing.
 
-You think to yourself: "Grandmother, no! What should I do now!?"
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.